translations/en.json

```diff
@@ -421,27 +426,33 @@
     "slides": {
       "welcome": {
         "title": "Welcome to PAAS City Parking App",
-        "text": "Through the application operated by the City of Bratislava, you can pay the hourly parking fee or use a bonus or visitor parking card."
+        "text": "Through the application operated by the City of Bratislava, you can pay the hourly parking fee or use a bonus or visitor parking card.",
+        "accessibilityLabel": "Go to Welcome to PAAS City Parking App"
```

Comment:
Hmm this could be composed directly in component like this? To prevent duplicate texts.
```
`Go to ${title}`
```